spec/factories/post_tags.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: post_tags
+#
+#  id         :bigint           not null, primary key
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#  post_id    :bigint
+#  tag_id     :bigint
+#
+# Indexes
+#
+#  index_post_tags_on_post_id             (post_id)
+#  index_post_tags_on_post_id_and_tag_id  (post_id,tag_id) UNIQUE
+#  index_post_tags_on_tag_id              (tag_id)
+#
+FactoryBot.define do
+  factory :post_tag do
+    tag { create(:tag) }
```

Comment:
I suggest to use `associate: :model`

https://devhints.io/factory_bot#associations